tidy3d/web/cli/develop.py

```diff
@@ -349,31 +351,45 @@ def commit_repository(repository_path, commit_message):
 def build_documentation(args=None):
     """Verifies and builds the documentation."""
     # Runs the documentation build from the poetry environment
-    # TODO update generic path management.
-    echo_and_run_subprocess(["poetry", "run", "python", "-m", "sphinx", "docs/", "_docs/"])
+    echo_and_check_subprocess(["poetry", "run", "python", "-m", "sphinx", "docs/", "_docs/"])
     return 0
 
 
 @develop.command(name="build-docs-pdf", help="Builds the sphinx documentation pdf.")
 def build_documentation_pdf(args=None):
     """Verifies and builds the documentation."""
     # Runs the documentation build from the poetry environment
-    # TODO update generic path management.
     echo_and_run_subprocess(
         ["poetry", "run", "python", "-m", "sphinx", "-M", "latexpdf", "docs/", "_pdf/"]
     )
     return 0
 
 
+@develop.command(
+    name="build-docs-remote-notebooks", help="Updates notebooks submodule and builds documentation."
+)
+def build_documentation_from_remote_notebooks(args=None):
+    """Updates notebooks submodule and builds documentation. Verifies and builds the documentation."""
+    # Runs the documentation build from the poetry environment
+    echo_and_check_subprocess(["git", "submodule", "update", "--remote"])
+    print("Notebook submodule updated from remote.")
+    echo_and_check_subprocess(["poetry", "run", "python", "-m", "sphinx", "docs/", "_docs/"])
+    return 0
+
+
 @develop.command(name="test-base", help="Tests the tidy3d base package.")
 def test_base_tidy3d(args=None):
-    """Verifies and builds the documentation."""
-    # Runs the documentation build from the poetry environment
-    # TODO update generic path management.
     echo_and_run_subprocess(["poetry", "run", "pytest", "-rA", "tests"])
     return 0
 
 
+@develop.command(name="test-notebooks", help="Tests the tidy3d notebooks.")
+def test_notebooks_tidy3d(args=None):
```

Comment:
any way (here or in a future PR) to add specific notebook names to run only? say I want to run the `ParameterSweep.ipynb` notebook (or maybe others) I can do `tidy3d develop test-notebooks ParameterSweep`? or something

Reply:
Yeah I think so through CLI arguments, I'll include it in my list for a future PR